pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -24,8 +24,12 @@ enum MulOperationType {
   IntTy,
 };
 
+using BlockArgumentSet = llvm::SmallPtrSet<mlir::BlockArgument, 8>;
+
 // Number of tensors for the matrix multiplication
 const unsigned kNumTensors = 3;
+// Number of searching indeces, i.e., M, N, K
```

Comment:
typo: indices

Reply:
Perhaps reword to: "Number of indices to search for (e.g. M, N, K)"

Reply:
Fixed